db-docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM postgres:latest
+
+ENV POSTGRES_USER alice
+ENV POSTGRES_PASSWORD password
+ENV POSTGRES_DB multisig
```

Comment:
Access these values from any env file or args. Don't expose sensitive information like passwords in code directly